src/secure-handlebars.js

```diff
@@ -52,9 +52,24 @@ function overrideHbsCreate() {
         return c.call(this, preprocess(template, options.strictMode), options);
     };
 
+    // make y refer to the default escape function
+    h.registerHelper('y', Handlebars.escapeExpression);
+
+    // don't escape SafeStrings, since they're already safe according to Handlebars
+    // Reference: https://github.com/wycats/handlebars.js/blob/master/lib/handlebars/utils.js#L63-L82
+    function safeStringCompatibleFilter (filterName) {
+        return function (s) {
+            // Unlike escapeExpression(), return s instead of s.toHTML() since downstream
+            //  filters of the same chain has to be disabled too.
+            //  Handlebars will invoke SafeString.toString() at last during data binding
+            return (s && s.toHTML) ? s : privFilters[filterName](s);
```

Comment:
what will privFilters[filterName](s) return if s is an object without toHTML api()? handlebars will do the force conversion in this line - https://github.com/wycats/handlebars.js/blob/master/lib/handlebars/utils.js#L77